package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "productly-layout",
+  "version": "1.0.0",
+  "description": "Project title:\r Layout of the adaptive landing page.",
+  "main": "index.js",
+  "dependencies": {
+    "gulp": "^4.0.2"
+  },
+  "devDependencies": {
+    "gulp-sass": "^5.0.0",
+    "gulp-sourcemaps": "^3.0.0",
+    "gulp-watch": "^5.0.1",
+    "i": "^0.3.6",
+    "sass": "^1.35.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Add some scripts to build and watch SCSS compilation. If you have scripts, you Donn't need to remember all CLI commands.